packages/components/src/components/post-language-option/post-language-option.scss

```
@@ -20,30 +20,32 @@ post-language-option {
     justify-content: center;
     height: 100%;
     width: 100%;
-    padding: var(--post-language-option-padding);
     border-radius: 2px;
     @include utilities-mx.focus-style;
   }
+
+  &:has([aria-current='true']),
+  &:has([aria-current='page']) {
+    :where([variant='menu']) & {
+      display: none;
+    }
+  }
```

If the aria-current option is hidden for both the menu and list variant, then it is more readable to add the `display: none` only once.

```suggestion
  &:has([aria-current]):not([aria-current='false']) {
    display: none;
  }
```